### ISSUE TITLE
Bad payload sent to lnbits

### ISSUE THREAD

**Issue by Bashy** [nbd-wtf/satdress#39]
I am already using a satdress on a domain that I don't manage with my lnbits,
I tried to make use satdress on my own domain but I get different payload sent to lnbits

On my satdress instance, LNbits returns a HTTP 400 bad request at the moment where invoice should be created, HTTP exchange below :
```
POST /api/v1/payments HTTP/1.1
Host: localhost:5000
User-Agent: Go-http-client/1.1
Content-Length: 148
Content-Type: application/json
X-Api-Key: [REDACTED]
Accept-Encoding: gzip

{"amount":1150,"out":false,"unhashed_description":"[[\"text/identifier\",\"ben@[REDACTED]\"],[\"text/plain\",\"Satoshis to ben@[REDACTED].\"]]"}


HTTP/1.1 400 Bad Request
date: Sat, 10 Sep 2022 22:11:22 GMT
server: uvicorn
content-length: 62
content-type: application/json

{"detail":"'unhashed_description' must be a valid hex string"}
```
Please note that I tried to put the remote API URL instead of the localhost one, the problem still remains.

Satdress logs that with the following line 
```
<nil> DBG invoice generation error="call to lnbits failed (400): {\"detail\":\"'unhashed_description' must be a valid hex string\"}" backend={"Cert":"","Host":"https://[REDACTED]","Key":"[REDACTED]"} bolt11= msatoshi=1500000
```

Here a working exchange with another satdress instance (I don't manage that other satdress instance)
```
POST /api/v1/payments HTTP/1.0
Host: [REDACTED]
X-Forwarded-Proto: https
X-Real-IP: [REDACTED]
X-Forwarded-For: [REDACTED]
Connection: close
Content-Length: 113
User-Agent: Go-http-client/1.1
Content-Type: application/json
X-Api-Key: [REDACTED]
Accept-Encoding: gzip

{"amount":1050,"out":false,"description_hash":"781eecd7a3a4aa7671dee05687a4d1e8ecc6e7bf2f02cca2c54b43aba369ef52"}


HTTP/1.1 201 Created
date: Sat, 10 Sep 2022 22:11:51 GMT
server: uvicorn
content-length: 603
content-type: application/json
connection: close
{"payment_hash":"1bf71b3220ed6fe71ae28705b2922f478cea731f0a7f9633da445beb14db57a9","payment_request":"[REDACTED]","checking_id":"[REDACTED]","lnurl_response":null}
```

It can be noticed that the payloads contain different objects `description_hash` for the one that works, and `unhashed_description` about the one that fails.

Using LNbits version 6f9ad06449739f4ad9ed22174e26cd7a6c59e1c2

**Comment by Bashy** [comment]
https://github.com/nbd-wtf/satdress/commit/37a104729cb7195f0794a0c5a845719d28382f82 
Can this commit be the source of the problem ?

**Comment by Bashy** [comment]
Reverting to https://github.com/nbd-wtf/satdress/commit/c1da2fb4eb34be598d77da93fdb282cecaa1857c solves the problem.
I was using master branch initially.

**Comment by fiatjaf** [comment]
Fixed in c5d0f65c88a506a07055d4d29c1cdc05661a1597.
